.github/workflows/build.yml

```diff
@@ -100,6 +100,12 @@ jobs:
       - name: Run Archetype Tests
         run: |
           ./mvnw -B clean install -Parchetype-test -pl :foundation-archetype
+      - name: Publish aggregated Cucumber report
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_dir: ./docs
```

Comment:
C: This looks like the wrong path. Shouldn't it be the target directory under the report aggregator project?